nav2_collision_monitor/src/velocity_polygon.cpp

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) 2022 Dexory
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "nav2_collision_monitor/velocity_polygon.hpp"
+#include "nav2_collision_monitor/polygon.hpp"
+
+#include "nav2_util/node_utils.hpp"
+
+
+namespace nav2_collision_monitor
+{
+
+VelocityPolygon::VelocityPolygon(
+  const nav2_util::LifecycleNode::WeakPtr & node,
+  const std::string & polygon_name,
+  const std::string & velocity_polygon_name)
+: node_(node), polygon_name_(polygon_name), velocity_polygon_name_(velocity_polygon_name)
+{
+  RCLCPP_INFO(logger_, "[%s]: Creating VelocityPolygon", velocity_polygon_name_.c_str());
+}
+
+
+VelocityPolygon::~VelocityPolygon()
+{
+  RCLCPP_INFO(logger_, "[%s]: Destroying VelocityPolygon", velocity_polygon_name_.c_str());
+  poly_.clear();
+}
+
+
+bool VelocityPolygon::getParameters()
```

Comment:
Some of the line extension breaks in this function for the getting / declaring of parameters are pretty awkward (e.x. `linear_max`), please take a look at this and make more readable and don't use excessive breaks. This is probably just ament_uncrustify's automatic response

Reply:
Yeah most of them are from `ament_uncrustify --reformat` (is there any dedicated formatter to use for this repo?). I have updated most of them manually now.